modules/component-store/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@ngrx/component-store",
-  "version": "19.2.1",
+  "version": "20.0.0",
```

These should be `20.0.0-beta.0` right?

```suggestion
  "version": "20.0.0-beta.0",
```

---

Oops, I was a bit too excited. Thanks!

---

Is this PR marked for merge?

---

CC: @markostanimirovic I would appreciate it if you could take a look at this or delegate it to someone who can. Thank you!